manifests/profile/server.pp

```diff
@@ -390,9 +395,9 @@
     }
 
     St2::Package::Install<| tag == 'st2::profile::server' |>
-    -> Ini_setting<| tag == 'st2::profile::server' |>
+    -> Ini_setting<| tag == 'st2::config' |>
     -> Exec['start st2']
```

Comment:
Also, eventually it would be nice if we could only restart a single service (e.g. for rbac we only need to restart st2api), but that's fine for now.